kpi/views/kpi_views.xml

```diff
@@ -0,0 +1,158 @@
+            <search string="KPI">
+                <group>
+                    <filter
+                        name="active"
+                        icon="terp-document-new"
+                        domain="[('active','=',True)]"
+                        string="Active"
+                        help="Only active KPIs are computed by the scheduler based on the periodicity configuration."
+                    />
+                    <separator orientation="vertical" />
+                    <field name="name" />
+                    <field name="category_id" />
+                    <field name="company_id" groups="base.group_multi_company" />
+                </group>
+                <newline />
+                <group expand="0" name="Group By...">
+                    <filter name="Category" context="{'group_by':'category_id'}" />
+                    <filter name="Type" context="{'group_by':'kpi_type'}" />
+                </group>
+            </search>
+        </field>
+    </record>
+    <record id="view_kpi_form" model="ir.ui.view">
+        <field name="name">kpi.form</field>
+        <field name="model">kpi</field>
+        <field name="arch" type="xml">
+            <form string="Key Performance Indicator">
+                <sheet>
+                    <group>
+                        <group>
+                            <field name="name" />
+                            <field name="category_id" />
+                        </group>
+                        <group>
+                            <field name="value" colspan="2" />
+                            <field name="active" colspan="2" />
+                            <field
+                                name="company_id"
+                                groups="base.group_multi_company"
+                            />
+                            <button
+                                name="compute_kpi_value"
+                                string="Compute KPI Now"
+                                colspan="2"
+                                type="object"
+                                groups="kpi.group_kpi_manager"
+                            />
+                        </group>
+                    </group>
+                    <notebook colspan="6">
+                        <page string="History">
+                            <field name="history_ids" readonly="1" nolabel="1" />
+                        </page>
+                        <page string="Computation" groups="kpi.group_kpi_manager">
+                            <group col="6">
+                                <field name="periodicity" colspan="3" />
+                                <field name="periodicity_uom" colspan="3" />
+                                <field name="next_execution_date" colspan="3" />
+                                <separator string="KPI Computation" colspan="6" />
+                                <newline />
+                                <field name="kpi_type" colspan="2" />
```

Comment:
It would be better to have colspan equal to "6" in order to have both kpi_type and kpi_code one bellow the other. 

With colspan = "2" , the label is in the same line as kpi_type and the editable field bellow, what makes the label confusing.